src/components/Posts/ActionType.tsx

```diff
@@ -78,42 +78,44 @@ export const FeedActionType = memo<FeedActionType>(function FeedActionType({ pos
     }, [post]);
 
     return (
-        <ClickableArea>
+        <ClickableArea className="w-full">
             {combined && !isPostPage ? (
                 <div className="mb-3 flex items-center space-x-2 text-[15px] text-secondary">
-                    <SparkIcon width={16} height={16} />
-                    <span className="flex items-center space-x-1">
-                        <strong>{profilesDescription}</strong> <span>{combinedDescription}</span>
+                    <SparkIcon width={16} height={16} className="flex-shrink-0" />
+                    <span className="flex max-w-[calc(100%-16px-8px)] items-center space-x-1">
```

Comment:
Instead of calculating, try `min-width:0`

```suggestion
                    <span className="flex min-w-0 items-center space-x-1">
```